Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -149,7 +149,7 @@ public override void Start()
             NodeConnectionParameters connectionParameters = this.connectionManager.Parameters;
             connectionParameters.IsRelay = !this.nodeSettings.ConfigReader.GetOrDefault("blocksonly", false);
             connectionParameters.TemplateBehaviors.Add(new ChainHeadersBehavior(this.chain, this.chainState));
-            connectionParameters.TemplateBehaviors.Add(new AddressManagerBehavior(this.addressManager));
+            connectionParameters.TemplateBehaviors.Add(new AddressManagerBehavior(this.addressManager) { PeersToDiscover = 10 });
```

Comment:
yes but when you do that you're reducing it to all the networks, not just StratisTestnet.
Is it intentional?

Reply:
yeah I think that's ok for now, there is a task to refactor the discovery away from being dependent on the connection loop (as its now)